apps/transport/lib/transport_web/controllers/page_controller.ex

```diff
@@ -56,6 +56,42 @@ defmodule TransportWeb.PageController do
     single_page(conn, %{"page" => "conditions"})
   end
 
+  def producteurs(conn, _params) do
+    conn
+    |> assign(:mailchimp_newsletter_url, Application.get_env(:transport, :mailchimp_newsletter_url))
+    |> render("producteurs.html")
+  end
+
```

Comment:
@fchabouis voilà une proposition d'implémentation (et j'ai mis des tests, mais en premier lieu pour m'aider à refactorer avec confiance).

J'y suis arrivé par itération, pas d'un coup, évidemment.

Tu me diras si ça te convient, et on peut en reparler.

Reply:
Yes c'est vachement mieux. Ca aurait été cool de le faire ensemble pour voir la démarche, mais ça m'apprend déjà beaucoup de choses de relire la version finale !

Reply:
Super :smile: hésite pas on pourra refaire le workflow en live ne serait-ce que 20 minutes, pour que je te montre comment j'aboutis à ça, à mon avis ça a de l'intérêt, préviens-moi si tu es intéressé et on calera ça!